src/test/java/org/broadinstitute/hellbender/utils/collections/NestedIntegerArrayUnitTest.java

```diff
@@ -57,7 +57,7 @@ public void testPutget1Key() throws Exception {
         arr.put("bozo", 1);
 
         Assert.assertEquals(2, arr.getAllLeaves().size());
-        Assert.assertEquals(Sets.newHashSet("fred", "bozo"), new HashSet<>(arr.getAllValues()));
+        Assert.assertEquals(Sets.newHashSet("fred", "bozo"), new LinkedHashSet<>(arr.getAllValues()));
```

Comment:
Can you audit the codebase for uses of `Sets.newHashSet()`, and determine whether they'll cause any issues (eg., by creating non-linked `HashSets`)